src/app/barter/components/barter-offer/barter-offer.component.html

```diff
@@ -0,0 +1,40 @@
+<div class="container-fluid">
+  <div class="d-flex justify-content-around">
+    <div class="ml-5 mt-5 item-card d-flex flex-column">
+      <mat-form-field appearance="fill">
+        <mat-label>Your product</mat-label>
+        <mat-select (selectionChange)="selectItem($event.value)" [(value)]="selectedItem">
+          <mat-option *ngFor="let item of myItems" [value]="item">
+            {{item.name}}
+          </mat-option>
+        </mat-select>
+      </mat-form-field>
+      <div>
+        <app-item #myItem [item]="selectedItem"></app-item>
+      </div>
+    </div>
+    <div class="d-flex justify-content-center align-items-center">
+      <button mdbBtn block="true" class="my-4 offer-deal-button text-white" (click)="offerDeal()">Offer Deal</button>
+    </div>
+
+    <div class="mr-5 item-card" style="margin-top: 7.5rem">
+      <app-item [item]="offerItem"></app-item>
+      <div class="text-center mt-4" style="border: 5px solid var(--main-theme-color);">
+        <div>
+          <h2>contact details</h2>
+        </div>
+        <div>
+          {{user.firstName.concat(' ', user.lastName)}}
```

Comment:
`${user.firstName} ${user.lastName}`

Reply:
what do you mean with the $? 
It's not observable or something it's just an entity- the user